rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -1295,6 +1295,358 @@ void testTxIsProcessableInLegacyVersion() {
         assertFalse(BridgeUtils.txIsProcessableInLegacyVersion(TxSenderAddressType.UNKNOWN, activations));
     }
 
+    @Test
+    void testIsMigrationTx_sending_funds_from_retired_p2sh_fed_to_active_p2sh_fed() {
+        NetworkParameters networkParameters = bridgeConstantsMainnet.getBtcParams();
+        Context btcContext = new Context(networkParameters);
+        when(activations.isActive(ConsensusRule.RSKIP201)).thenReturn(true);
+
+        List<BtcECKey> retiredFederationKeys = Stream.of(
+            BtcECKey.fromPrivate(Hex.decode("fc01")),
```

Comment:
Isn't it better to actually use a random hex string with the length of an actual private key? I don't think 'fc01' is a good example of a private key here.

Reply:
I would say yes if this test were related to testing something with signing/or with signature. But, this is not the case. It's just testing that for a given btc transaction sending funds to the fed standardP2shScript of each type of Federation, the tx is processed correctly by the bridge.